packages/editor/src/components/inserter/menu.js

```diff
@@ -205,6 +205,15 @@ export class InserterMenu extends Component {
 			itemsPerCategory,
 			openPanels,
 		} );
+
+		const resultCount = Object.keys( itemsPerCategory ).reduce( ( memo, curr ) => memo + itemsPerCategory[ curr ].length, 0 );
```

Comment:
I'll fix this, but for future reference: please use full words for variables whenever possible. It's easier to read `current` than `curr`. I assume this should be `current`, right? 😄

Reply:
Thanks @tofumatt, I'll make sure to read over the documentation to make sure I follow the standards going forward! 👍